lib/metadata/templates/title.ts

```diff
@@ -51,5 +51,8 @@ const TEMPLATE_MAP: Record<Route['pathname'], string> = {
 export function make(pathname: Route['pathname']) {
   const template = TEMPLATE_MAP[pathname];
 
-  return `%network_name% ${ template }`;
+  const capitalizedTemplate = template.charAt(0).toUpperCase() + template.slice(1);
+  const networkWithTemplate = `%network_name% Block Explorer${ capitalizedTemplate ? ` - ${ capitalizedTemplate }` : '' }`;
+
+  return networkWithTemplate + ' | Horizen EON';
```

Comment:
this field will be dynamic with the sidechain environment (pre-testnet, testnet, etc.), right?